spec/smoke_tests/jobseekers_can_view_homepage_spec.rb

```diff
@@ -1,9 +1,15 @@
 require "capybara/rspec"
 require "i18n_helper"
 
+PRODUCTION_DOMAIN = "teaching-vacancies.service.gov.uk".freeze
+
 RSpec.describe "Page availability", js: true, smoke_test: true do
   context "Jobseeker visits vacancy page" do
-    let(:smoke_test_domain) { ENV.fetch("SMOKE_TEST_DOMAIN", "teaching-vacancies.service.gov.uk") }
+    let(:smoke_test_domain) do
+      smoke_test_domain = ENV.fetch("SMOKE_TEST_DOMAIN", "")
```

Comment:
This could all be

```ruby
let(:smoke_test_domain) { ENV.fetch("SMOKE_TEST_DOMAIN").presence ||  "teaching-vacancies.service.gov.uk" }
```

Reply:
we tried it and .presence is only available in rails, so we would need to require it. is it worth it?
also it fails before .presence with `KeyError: key not found: "SMOKE_TEST_DOMAIN"`

Reply:
I'll merge this for now as we need smoke tests, let's discuss later a more elegant option